src/js/helper.js

```diff
@@ -0,0 +1,15 @@
+function GUI_checkOperatingSystem() {
+    if (navigator.appVersion.indexOf("Win") !== -1) {
```

Comment:
https://github.com/betaflight/betaflight-configurator/blob/c8d0490b2ffa66615ea37da0e9a7282af8d01077/src/js/gui.js#L77

Reply:
removed helper.js, the reference conflict got removed and its not needed in port_handler anymore.